core/metadata/test/LoadUnit.test.ts

```
@@ -1,20 +1,34 @@
-import { BizManager } from './fixtures/modules/app-multi-inject-multi/app/modules/bar/BizManager';
-import { Secret } from './fixtures/modules/app-multi-inject-multi/app/modules/foo/Secret';
+// import { App } from './fixtures/modules/app-multi-inject-multi/app/modules/app/App';
+// import { App2 } from './fixtures/modules/app-multi-inject-multi/app/modules/app2/App';
+// import { BizManager } from './fixtures/modules/app-multi-inject-multi/app/modules/bar/BizManager';
+// import { Secret } from './fixtures/modules/app-multi-inject-multi/app/modules/foo/Secret';
+import { buildGlobalGraph } from './fixtures/LoaderUtil';
 
 
 describe('test/LoadUnit/LoadUnit.test.ts', () => {
+  beforeEach(() => {
+    delete GlobalGraph.instance;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)